src/services/appServiceWebApp/data.ts

```diff
@@ -63,17 +63,15 @@ export default async ({
             const webAppsIterable: PagedAsyncIterableIterator<Site> =
               client.appServicePlans.listWebApps(resourceGroupId, name)
             for await (const webApp of webAppsIterable) {
-              if (webApp) {
-                const { location, extendedLocation, identity, tags, ...rest } =
-                  webApp
-                const region = lowerCaseLocation(location)
-                return {
-                  ...rest,
-                  appServicePlanId: appService.id,
-                  region,
-                  resourceGroupId,
-                  Tags: tags || {},
-                }
+              const { location, extendedLocation, identity, tags, ...rest } =
```

Comment:
is there a case where webApp is undefined? Do we need something like
```
if (!webApp) { continue; }
```

Reply:
Yes, we need to ask if `webApp` has a truth value. I'm missing a commit where I added the validation